### PR TITLE
[Outlook] Requirement set 1.10

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Related PRs: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52860, https://github.com/OfficeDev/office-js-snippets/pull/535

### CODE REVIEW

File: docs/docs-ref-autogen/outlook_1_10/outlook/office.attachmentcontent.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    fullName: format
+    summary: >-
+      The string format to use for an attachment's content.
+
+
+      For file attachments, the formatting is a base64-encoded string.
+
+
+      For item attachments that represent messages and were attached by drag-and-drop or "Attach Item", the formatting
+      is a string representing an .eml formatted file. **Important**: If a message item was attached by drag-and-drop in
```
Comment (lindalu-MSFT) [inline docs/docs-ref-autogen/outlook_1_10/outlook/office.attachmentcontent.yml:47]: Shouldn't the alert be on it's own line?

File: docs/docs-ref-autogen/outlook_1_10/outlook/office.attachmentcontent.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      Outlook on the web, then `getAttachmentContentAsync` throws an error.
+
+
+      For item attachments that represent calendar items and were attached by drag-and-drop or "Attach Item", the
+      formatting is a string representing an .icalendar file. **Important**: If a calendar item was attached by
```
Comment (lindalu-MSFT) [inline docs/docs-ref-autogen/outlook_1_10/outlook/office.attachmentcontent.yml:52]: ditto

**Review by lindalu-MSFT — COMMENTED** [review]
Hi, I admit I didn't do a very thorough review of this one. I'm having a hard time reading the YAML so I went to the preview docs and spot checked a few pages. It looks good, I noticed a couple niggly things but I'm not sure they can be fixed in YAML.

File: docs/docs-ref-autogen/outlook_1_10/outlook/office.appointment.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+### YamlMime:TSType
+name: Office.Appointment
+uid: 'outlook!Office.Appointment:interface'
+package: outlook!
+fullName: Office.Appointment
+summary: >-
+  The subclass of [Item](xref:outlook!Office.Item:interface) dealing with appointments.
+
+
+  **Important**: This is an internal Outlook object, not directly exposed through existing interfaces. You should treat
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/outlook_1_10/outlook/office.appointment.yml:10]: @lindalu-MSFT No, we can't

**Pull request merged**